spotifyPlayer.py

```diff
@@ -65,7 +65,7 @@ def playMP3(fileName):
 
 		# Removes the MP3 file once song is done playing
 		def checkPlayStatus(fileName):
-			while pygame.mixer.music.get_busy():
+			while mixer.music.get_busy():
```

Comment:
Does this return `true` if you call`mixer.music.pause()`?

Reply:
Yes. `pygame.mixer.music.pause()` doesn't affect the state of `pygame.mixer.music.get_busy()`